processing/src/main/java/org/apache/druid/query/aggregation/last/DoubleLastAggregatorFactory.java

```diff
@@ -230,7 +248,7 @@ public String getTypeName()
   @Override
   public int getMaxIntermediateSize()
   {
-    return Long.BYTES + Double.BYTES;
+    return Long.BYTES + Double.BYTES + 1;
```

Comment:
`Byte.BYTES`. Or perhaps adding a new variable `NULL_SIZE`?

Reply:
it seems unlikely to me that the number of bytes in a byte is not going to change, but ok :p